client/lib/plans/features-list.tsx

```diff
@@ -1707,6 +1708,10 @@ export const FEATURES_LIST: FeatureList = {
 		getSlug: () => FEATURE_NEWSLETTER_IMPORT_SUBSCRIBERS_FREE,
 		getTitle: () => i18n.translate( 'Import up to 100 subscribers' ),
 	},
+	[ FEATURE_PAYMENT_TRANSACTION_FEES_10 ]: {
+		getSlug: () => FEATURE_PAYMENT_TRANSACTION_FEES_10,
+		getTitle: () => i18n.translate( '10% transaction fee for payments + Stripe fees' ),
```

Comment:
I copied this from lines around for 4% and 8%, but I think this would've been better as % being a placeholder, so that we don't need to translate the same sentence multiple times, and doing changes  in future will need less translation coordination.